visidata/loaders/mysql.py

```diff
@@ -0,0 +1,114 @@
+from visidata import *
+
+__all__ = ['openurl_mysql', 'MyTable', 'MyTablesSheet']
+
+def codeToType(type_code, colname):
+    import MySQLdb as mysql
+
+    if type_code == mysql.NUMBER:
```

Comment:
Um, you mean I rather should check for some of these types?

https://github.com/PyMySQL/mysqlclient-python/blob/4c1950148ba10e23017bd5190f9da6bdb347e6b3/MySQLdb/constants/FIELD_TYPE.py#L7-L40

Reply:
I checked again, and I think `mysql.NUMBER` is the correct way to evaluate `type_code`; it actually is a set of type codes looking like this:

```py
DBAPISet({0, 1, 3, 4, 5, 8, 9, 13, 246})
```

Reply:
Yes, sorry, I meant that some numbers are `int` and some numbers are `float` and we should distinguish them like MySQL does.  (so that 54.8434 can be a float but 54 can be an int).

Reply:
I am still not sure how to proceed: should I check and return `int` for all of those types, then?

* DECIMAL = 0
* TINY = 1
* SHORT = 2
* LONG = 3
* LONGLONG = 8
* INT24 = 9
* NEWDECIMAL = 246

Reply:
Yes, though it seems like we should return `float` for FLOAT, DOUBLE, DECIMAL, and NEWDECIMAL?

Reply:
Aha! I just went along with the Postgres loader as template, which only returns `INTEGER` and `STRING`, too… let me see.

Reply:
Like this? 56bbee88f2147c3b47ba772d8d1329e37199b347